### DISCUSSION TITLE
Jotai Devtools relies on redux devtools?

### DISCUSSION THREAD

**Discussion by npearson72** [jotaijs/jotai-devtools#90] [Q&A] [answered]
I'm noticing the following warning: `jotai-devtools > @redux-devtools/extension@3.2.5" has unmet peer dependency "redux@^3.1.0 || ^4.0.0".`

Would love to understand the situation here. Anyone know why redux devtools are needed?

**Comment by arjunvegda** [comment, marked as answer]
Hi @npearson72 

We provide hooks that let you use the Redux DevTools extension to debug your Jotai Atoms 😁 

See: https://jotai.org/docs/tools/devtools#useatomdevtools

**Reply by npearson72** [reply]
Interesting. Thank you.
